src/auctionManager.js

```diff
@@ -0,0 +1,433 @@
+const adaptermanager = require('./adaptermanager');
```

Comment:
You've got a circular dependency here. The `auctionManager` imports the `adapterManager`, but the `adapterManager` has a `callBids(auction, cbTimeout)` method. Circular dependencies cause a bunch of problems (we can talk if you're interested)... so one of these two should not exist.

IMO `auctionManager` should depend on `adapterManager`, because you can't do an Auction without Adapters to call. An Adapter could be used outside of the context of an Auction, though.